tests/mdast_serde.rs

```diff
@@ -0,0 +1,10 @@
+use markdown::mdast::Node;
+
+#[cfg_attr(feature = "serde", test)]
+fn test_serde() {
+    let markdown = "This is a **test**\n\n> quote content\n> continue";
+    let tree = markdown::to_mdast(&markdown, &markdown::ParseOptions::default()).unwrap();
+    let json = serde_json::to_string(&tree).unwrap();
+    let tree2: Node = serde_json::from_str(&json).unwrap();
+    assert!(tree == tree2);
+}
```

Comment:
Thanks. Can you add one more, simple, test, to check the *actual* JSON?

Because JSON *allows* duplicate fields. It’s just that currently, perhaps, serde doesn’t parse that. But that might change in the future.

I’d like to know that a *string* of JSON does not include duplicate fields, instead of whether Serde can parse its own output! (I’d assume it always would do that?)

Reply:
done